packages/button/src/__snapshots__/Button.test.tsx.snap

```diff
@@ -12,6 +13,8 @@ exports[`renders correctly 1`] = `
   background: transparent;
   font-family: inherit;
   font-weight: 700;
+  -webkit-text-decoration: none;
```

Comment:
Don't we have any prefixer for this?

Reply:
It's a test snapshot, tests see prefixed code:)